python/hsfs/core/delta_engine.py

```diff
@@ -109,9 +138,28 @@ def _setup_delta_read_opts(self, delta_fg_alias, read_options):
             delta_options = {
                 self.DELTA_QUERY_TIME_TRAVEL_AS_OF_INSTANT: _delta_commit_end_time,
             }
+        elif delta_fg_alias.left_feature_group_start_timestamp is not None:
+            # change data feed query with start and end time
+            _delta_commit_start_time = util.get_delta_datestr_from_timestamp(
+                delta_fg_alias.left_feature_group_start_timestamp
+            )
+            _delta_commit_end_time = util.get_delta_datestr_from_timestamp(
```

Comment:
this should move after `if delta_fg_alias.left_feature_group_end_timestamp is not None:`